Ecommerce/index.html

```diff
@@ -1,32 +1,76 @@
 
-<!DOCTYPE html>
-<html><head><title>Task1</title></head>
-<body>
-<h1>Welcome to My Shop</h1>
-<img src="shp.png">
-<ul>
-<li><a href="#">About Us</a></li>
-<li><a href="#">Products</a></li>
-<li><a href="#">Contact Us</a></li>
-</ul>
-<h2>About Us</h2>
-Welcome to My Shop! We are a one -stop destination for all your shopping needs. Our mission is to provide you with the best products and exceptional customer service.
-<h2>Products</h2>
-Explore our wide range of products, including:
-<ul>
-<li>Electronics</li><li>Clothing</li><li>Home Decor</li><li>Toys</li>
-</ul>
-Find perfect items for yourself, or as gift for your loved ones.
-<h2>Contact Us</h2>
-If you have any questions or need assistance, please don't hesitate to reach out to us using the contact form below:
-<form action="#">
-<label for="name">Name:</label>
-<input type="text" id="name"><br>
-<label for="email">Email:</label>
-<input type="text" id="email"><br>
-<label for="message">Message:<br></label>
-<textarea id="message" name="message" rows="4" cols="50"></textarea><br>
-<button> Submit</button>
-<p>&copy; 2023 My Shop. All Right Reserved.</p>
+<html>
+    <head>
+        <title>Implying Layputs to My Shop</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>
+            .top{
+                background-color:#4D133F;
+                width:100%;
+                height:8rem;

```

Comment:
There is an extra line with only whitespace in this line, it should be removed for cleaner code organization